website/project/views/contributor.py

```diff
@@ -344,8 +344,8 @@ def project_remove_contributor(auth, **kwargs):
             raise HTTPError(http.BAD_REQUEST, data={
                 'message_long': 'Could not remove contributor.'})
 
-        # On parent node, if user has removed herself from project, alert; redirect to user
-        # dashboard if node is private, else node dashboard
+        # On parent node, if user has removed herself from project, alert; redirect to
+        # user's my projects page if node is private, else node dashboard
```

Comment:
"node dashboard" -> "node summary" (I know you didn't do that, but since you're there, let's get the whole sentence consistent with modern usage)